node-setup/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM to-be-replaced-by-local-ref/base:ubuntu
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends xfsprogs && \
+    apt-get clean
```

Comment:
you also need to clean the apt lists
https://github.com/scylladb/scylla-operator-images/blob/2a450f9/base/ubuntu/Dockerfile#L10
looks like I forgot to do that in the golang one

Reply:
fixed, thanks